### ISSUE TITLE
Layout/FirstArgumentIndentation is off-by-[1|2] when using splats with chained methods

### ISSUE THREAD

**Issue by pdobb** [rubocop-hq/rubocop#9155]
This is a follow-up to https://github.com/rubocop-hq/rubocop/issues/9115.

There is still/another issue in rubocop 1.5.1 with splatting and the Layout/MultilineMethodCallIndentation cop with `EnforcedStyle: consistent_relative_to_receiver`.

--------

## Expected behavior

Both of these examples should be valid:

```ruby
# Valid :)
[
  value1,
  *my_method(
    arg1)
]

# Invalid :(
[
  *my_object
    .my_chained_method(
      arg1)
]
```

## Actual behavior

Method chains appear to require an extra space to offset the splat operator.

```ruby
# This is considered invalid, when it shouldn't be
[
  *my_object
    .my_chained_method1(
      arg1)
]

C: Layout/MultilineMethodCallIndentation: Indent .my_chained_method 2 spaces more than my_object on line 4.
    .my_chained_method1(
    ^^^^^^^^^^^^^^^^^^

# And will autocorrect to this, when it shouldn't (Note the extra space before `.my_chained_method(`
[
  *my_object
     .my_chained_method1(
      arg1)
]

```

Also:

```ruby
# This is considered invalid, when it shouldn't be
{
  **my_object
    .my_chained_method2(
      arg1)
}

C Layout/MultilineMethodCallIndentation: Indent .my_chained_method 2 spaces more than my_object on line 4.
    .my_chained_method2(
    ^^^^^^^^^^^^^^^^^^

# And will autocorrect to this, when it shouldn't (Note the extra 2 spaces before `.my_chained_method(`
{
  **my_object
      .my_chained_method2(
      arg1)
}
```

## Steps to reproduce the problem

Add to temp file and lint with `rubocop --only Layout/MultilineMethodCallIndentation temp.rb`

```ruby
[
  *my_object
    .my_chained_method(
      arg1)
]

{
  **my_object
    .my_chained_method(
      arg1)
}
```

Note, that my configuration is:

> Layout/FirstArgumentIndentation:
>  EnforcedStyle: consistent_relative_to_receiver

## RuboCop version

```
$ rubocop -V
1.5.1 (using Parser 2.7.2.0, rubocop-ast 1.3.0, running on ruby 2.6.6 x86_64-darwin19)
  - rubocop-performance 1.9.0
  - rubocop-rails 2.8.1
  - rubocop-rake 0.5.1
```
